app/services/shopify_import/products/create.rb

```diff
@@ -38,17 +64,17 @@ def product_tags
         parser.product_tags
       end
 
+      def option_types_data
+        parser.option_types
+      end
+
       def parser
         @parser ||= DataParsers::BaseData.new(shopify_product)
       end
 
       def shopify_product
         @shopify_product ||= ShopifyAPI::Product.new(JSON.parse(@shopify_data_feed.data_feed))
       end
-
-      def shipping_category
```

Comment:
This shows how adding a code before it's needed is unnecessary and makes code reviewing harder.